test/handle_access_token_response_test.dart

```diff
@@ -47,6 +47,12 @@ void main() {
       expect(() => handleError(headers: {
         'content-type': 'text/plain'
       }), throwsFormatException);
+    }, skip: 'text/plain content-type is accepted now');
```

Comment:
Well, not that `text/plain` is accepted, but that you could manually accept it using `getParameters`. I'll definitely remove it, though, since it's not necessary.